cdk/src/open-data-platform/data-plane/schema/schema.sql

```diff
@@ -95,22 +95,44 @@ CREATE INDEX IF NOT EXISTS census_state_geo_id_index ON demographics (state_cens
 CREATE INDEX IF NOT EXISTS census_county_geo_id_index ON demographics (county_census_geo_id);
 CREATE INDEX IF NOT EXISTS geom_index ON demographics USING GIST (geom);
 
+--- Tables related to U.S. demographic information.
+
+-- TODO(breuch): Add insert statement once the zipcode -> demographics
+
+CREATE TABLE IF NOT EXISTS aggregate_us_demographics
+(
+    census_geo_id                varchar(255) NOT NULL,
+    geo_type                     varchar(255) NOT NULL,
+    name                         varchar(255) NOT NULL,
+    average_home_age             real,
+    average_income_level         real,
+    average_social_vulnerability real,
+    black_population             real,
```

Comment:
I don't think you need lines 110-115

Reply:
SG - wasn't sure of best way to migrate over while we don't have no data, but I'll assume we want this to look as close to end state as possible.